charts/argo-cd/templates/argocd-server/deployment.yaml

```diff
@@ -299,6 +299,12 @@ spec:
                 name: argocd-cmd-params-cm
                 key: server.enable.proxy.extension
                 optional: true
+          - name: ARGOCD_API_CONTENT_TYPES
```

Comment:
added in v2.9.6 change

Reply:
✏️ 
https://github.com/argoproj/argo-cd/blob/v2.9.6/manifests/base/server/argocd-server-deployment.yaml#L230-L235